### PR TITLE
docs(serverless): renamed mCPU to mVCPU

### PR DESCRIPTION
**Pull request by bene2k1**
### Description
Serverless Containers & Functions: Renamed mCPU to mVCPU

### CODE REVIEW

File: styles/Vocab/Scaleway/accept.txt [hunk 1]
```diff
@@ -44,7 +44,7 @@ KubeConfig
 MiniKube
 ReplicaSet
 Docker
-mCPU
+mVPCU
```
Comment (M-Astrid-M) [inline styles/Vocab/Scaleway/accept.txt:47]: mVCPU

File: faq/serverless-containers.mdx [hunk 2]
```diff
@@ -62,7 +62,7 @@ For now, these two metrics are divided into 5 tiers:
     * *Billed resources:* 3 750 000 - 400 000 = 3 350 000 GB/s
     * *Cost:* 3 350 000 * €0.0000010 = **€3,35**
 * **vCPU consumption**
-    * *Allocated vCPU conversion:* 70mCPU = 0.070 vCPU
+    * *Allocated vCPU conversion:* 70mVPCU = 0.070 vCPU
```
Comment (M-Astrid-M) [inline faq/serverless-containers.mdx:65]: mVCPU

File: compute/containers/concepts.mdx [hunk 3]
```diff
@@ -76,10 +76,10 @@ JWT (JSON Web Token) is an access token you can create from the console or API t
 
 <Concept>
 
-## mCPU
+## mVPCU
```
Comment (M-Astrid-M) [inline compute/containers/concepts.mdx:79]: mVCPU

File: changelog/serverless/october2022/2022-10-13-containers-added-memory-cpu.mdx [hunk 4]
```diff
@@ -9,4 +9,4 @@ category: serverless
 product: containers
 ---
 
-New tiers for Serverless Containers (3GB/1680 mCPU & 4GB/2240 mCPU) available.
\ No newline at end of file
+New tiers for Serverless Containers (3GB/1680 mVPCU & 4GB/2240 mVPCU) available.
```
Comment (M-Astrid-M) [inline changelog/serverless/october2022/2022-10-13-containers-added-memory-cpu.mdx:12]: mVCPU

File: tutorials/deploy-umami-on-serverless-containers/index.mdx [hunk 5]
```diff
@@ -74,7 +74,7 @@ The tool collects only the metrics relevant to you and displays everything on a
 2. Click **+ Deploy a Container**. The container deployment wizard displays.
 3. Choose the namespace you pushed the image to and select the `umami:postgresql-latest` image. Set the value for the port to `3000`.
 4. Enter a name for your container. Optionally, you can enter a container description. 
-5. Select the resources you want to allocate to the container. The resources to allocate depend on the intended load of your application. In this tutorial we use **128 MB - 70 mCPU**.
+5. Select the resources you want to allocate to the container. The resources to allocate depend on the intended load of your application. In this tutorial we use **128 MB - 70 mVPCU**.
```
Comment (M-Astrid-M) [inline tutorials/deploy-umami-on-serverless-containers/index.mdx:77]: mVCPU

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**